Libraries/LibWeb/CSS/CSSTransition.cpp

```diff
@@ -74,10 +74,14 @@ Optional<int> CSSTransition::class_specific_composite_order(GC::Ref<Animations::
     if (m_transition_generation != other->m_transition_generation)
         return m_transition_generation - other->m_transition_generation;
 
-    // FIXME:
     // 5. Otherwise, sort A and B in ascending order by the Unicode codepoints that make up the expanded transition
     //    property name of each transition (i.e. without attempting case conversion and such that ‘-moz-column-width’
     //    sorts before ‘column-width’).
+    if (transition_property() > other->transition_property())
+        return 1;
+    else
+        return -1;
+
     return {};
```

Comment:
This `return {};` is dead code. But why not just `return transition_property().compare(other->transition_property())`?

Reply:
I never mentioned `>` though?